src/components/VideoPlayer/VideoTimeline/VideoTimeline.module.scss

```
@@ -90,7 +92,7 @@ $video-player-controls-track-color: rgba($video-player-controls-bg-color, 0.3) !
     &::-webkit-slider-thumb {
       -webkit-appearance: none;
       @include thumb;
-      margin-top: -$video-player-controls-thumb-size/4;
+      margin-top: math.div($video-player-controls-thumb-size, 4);
```

Maybe removing the `-` was intended?

```suggestion
      margin-top: math.div(-$video-player-controls-thumb-size, 4);
```